config_hub/pretrain/debug.yaml

```diff
@@ -49,7 +49,7 @@ train:
   max_seq_length:
 
   # Whether to tie the embedding weights with the language modelling head weights. (type: Optional[bool], default: False)
```

Comment:
> modelling head weights

Small thing, but since we use American over British English, we could say "modeling" instead of "modelling". Or even better maybe just "model heads"